### ISSUE TITLE
Cannot use createTexture in mac

### ISSUE THREAD

**Issue by remorses** [ziglibs/zgl#56]
Mac OpenGl version is stuck on OpenGL 4.1

The following line is using a function that requires 4.5 ([glCreateTextures](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glCreateTextures.xhtml))
https://github.com/ziglibs/zgl/blob/b2c03796ecbe54faa09370ce94d6658c7ee7d315/zgl.zig#L1357

Does this mean that zgl cannot run on mac systems? Is there any workaraound?

**Comment by remorses** [comment]
I am a total OpenGL noob that is trying to run this [project](https://github.com/N00byEdge/voxig) on mac and getting this error

There is probably an alternative function to create textures that work on previous opengl version right? So probably the problem is that specific code and not zgl

**Comment by ikskuh** [comment]
Yeah, zgl was built with OpenGL 4.5 in mind right now. I don't intent to change this soon, but maybe enable a API wrapper to allow the use of `GL_ARB_direct_state_access` instead.

But i have to be honest: I don't use `zgl` anymore personally and keep the project in maintainance mode. Feel free to figure out a way and PR it though!

**Comment by quantumedbox** [comment]
> I am a total OpenGL noob that is trying to run this [project](https://github.com/N00byEdge/voxig) on mac and getting this error
> 
> There is probably an alternative function to create textures that work on previous opengl version right? So probably the problem is that specific code and not zgl

Yes, you can use `glGenTextures` instead, which is wrapped under `genTexture` and available from version 2.0. I personally have no issues using extended OpenGL 2.1 with zgl. Probably too late of an answer, but somebody might be looking at it in the future, so.
